test/integration/pipeline/multisig_tx_pipeline_test.cpp

```diff
@@ -157,15 +158,28 @@ TEST_F(MstPipelineTest, OnePeerSendsTest) {
       };
 
   auto &mst_itf = prepareMstItf();
-  mst_itf.sendTx(complete(tx, kUserKeypair), check_mst_pending_tx_status)
+  auto mst_tx = complete(tx, kUserKeypair);
+  mst_itf.sendTx(mst_tx, check_mst_pending_tx_status)
       .sendTx(complete(tx, signatories[0]), check_mst_pending_tx_status)
       .sendTx(complete(tx, signatories[1]),
-              check_enough_signatures_collected_tx_status)
-      .skipProposal()
-      .skipVerifiedProposal()
-      .checkBlock([](auto &proposal) {
-        ASSERT_EQ(proposal->transactions().size(), 1);
-      });
+              check_enough_signatures_collected_tx_status);
+
+  // wait until multisig transaction is appeared
```

Comment:
Why we need `while` here? How can we get proposal without these two transactions above?

Reply:
There are a few reasons for do that:
* now we can't relay on order of completed transactions from MST
* if there will be some MST regression where MST processor shares not completed transactions we will catch it